src/vm/typehashingalgorithms.h

```diff
@@ -9,19 +9,26 @@
 #pragma once
 #include <stdlib.h>
 
+// DIFFERENT FROM CORERT: Hashing code currently different from CoreRT
+
 //
 // Returns the hashcode value of the 'src' string
 //
-inline static int ComputeNameHashCode(const SString &src)
+inline static DWORD ComputeNameHashCode(LPCUTF8 src)
```

Comment:
Why did you have to change this to DWORD? I think it is better to keep it as int.